src/constants/document-file-type.constant.ts

```diff
@@ -0,0 +1,18 @@
+export const DOCUMENT_FILE_TYPE = {
```

Comment:
`ALLOWED_DOCUMENT_FILE_TYPE` might be a bit better to emphasise that changing this object changes the document types that the client is allowed to use (rather than just being for any file type you want to include in the code) - what do you think?

Reply:
Yes, that sounds like a good idea